core/interfaces/i_parameter_model.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * The interface for the data model of a procedure parameter.
+ *
+ * @namespace Blockly.IParameterModel
+ */
+import * as goog from '../../closure/goog/goog.js';
+goog.declareModuleId('Blockly.IParameterModel');
+
+import {VariableModel} from '../variable_model.js';
+
+/**
+ * A data model for a procedure.
+ */
+export interface IParameterModel {
```

Comment:
I note that [the JS style guide still prohibits Hungarian notation](https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers).  I'd be interested to know what the TS Stylist's opinion about this is.

Reply:
Yeee, but we've discussed this several times, and decided to go with it for consistency for now.